docs/application/native/guides/multimedia/media-controller.md

```diff
@@ -187,21 +327,252 @@ To process the received command on the server side:
 
 3. Register the callback:
 
-   - To register a callback for playback state change commands, use the `mc_server_set_playback_state_command_received_cb()` function.
-   - To register a callback for a custom command, use the `mc_server_set_custom_command_received_cb()` function.
+   - To register a callback for playback state change commands, use the `mc_server_set_playback_state_command_received_cb()`.
+   - To register a callback for a custom command, use the `mc_server_set_custom_command_received_cb()`.
 
    For example, to register a callback for playback state change commands:
 
    ```
    ret = mc_server_set_playback_state_command_received_cb(g_mc_server, command_received_cb, NULL);
    ```
 
-4. When no longer needed, destroy the media controller server handle using the `mc_server_destroy()` function:
+4. Destroy the media controller server handle using the `mc_server_destroy()`, when media controller server handle is no longer needed:
 
    ```
    mc_server_destroy(g_server_h);
    ```
-<a name="serverstate"></a>
+
+
+## Sending and Processing Commands To Receive Replies
```

Comment:
Please change "To" to "to". It doesn't need to be in capital letter.

Reply:
It's done.